### PR TITLE
clipboard-jh: init at 0.3.2

### PR DESCRIPTION
**Pull request by dit7ya**
###### Description of changes

[Slackadays/Clipboard: 📋 Cut, copy, and paste anything, anywhere, all from the terminal.](https://github.com/Slackadays/Clipboard)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/clipboard/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    sha256 = "sha256-gFYw4iqb7WzJYQihmSe/VC0TQo+RV5SLTkqXJiDVISg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    xorg.libX11
+    wayland-protocols
+    wayland
+    libffi
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/clipboard/default.nix:29]: Please split the packages ready for cross compilation.

File: pkgs/tools/misc/clipboard/default.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "-DCMAKE_BUILD_TYPE='MinSizeRel'"
+    "-Wno-dev"
+    "-DINSTALL_PREFIX=${placeholder "out"}"
+  ];
+
+  meta = with lib; {
+    description = "Cut, copy, and paste anything, anywhere, all from the terminal";
+    homepage = "https://github.com/Slackadays/clipboard";
+    license = licenses.gpl3Only;
+    mainProgram = "clipboard";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/clipboard/default.nix:42]:
```suggestion
```
mainProgram defaults to pname

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by dit7ya** [comment]
Not sure how to rename this.

**Comment by o69mar** [comment]
> Not sure how to rename this.

0.3.2 when

**Comment by dit7ya** [comment]
> > Not sure how to rename this.
> 
> 0.3.2 when

@o69mar do you have any suggestions for renaming this package to something else as `clipboard` already exists in nixpkgs?

**Comment by o69mar** [comment]
> > > Not sure how to rename this.
> > 
> > 0.3.2 when
> 
> @o69mar do you have any suggestions for renaming this package to something else as `clipboard` already exists in nixpkgs?

idk man, also slackadays is asking me for a nix flake, but idk how to code one, i tried it before but i got errors over and over again.

**Comment by o69mar** [comment]
> > > Not sure how to rename this.
> > 
> > 
> > 0.3.2 when
> 
> @o69mar do you have any suggestions for renaming this package to something else as `clipboard` already exists in nixpkgs?

@dit7ya i created a nix flake for clipboard, this PR shouldn't be needed now.

**Comment by Slackadays** [comment]
Rename it `clopboard-jh`

**Comment by Slackadays** [comment]
@SuperSandro2000 Any progress regarding this? It looks like the renaming and other issues have been fixed.

**Comment by SuperSandro2000** [comment]
@ofborg build clipboard-jh

**Comment by o69mar** [comment]
> @ofborg build clipboard-jh

@Slackadays big W

**Pull request merged**